sdk/cosmosdb/cosmos/test/internal/session.spec.ts

```diff
@@ -66,7 +66,7 @@ describe("New session token", function() {
       resourceId: "1"
     });
     assert.equal(responseToken, token);
-    assert.equal(responseToken, rqContext.headers["x-ms-session-token"]);
+    assert.equal(responseToken, rqContext.headers?.["x-ms-session-token"]);
```

Comment:
does this work with the period after the `?` sign? 

I would've thought you either had to index directly or do `.get(["x-ms-session-token])`